### DISCUSSION TITLE
PYG Latest

### DISCUSSION THREAD

**Discussion by shjiang-yang** [pyg-team/pytorch_geometric#3382] [Q&A] [answered]
I learned in the latest document that GATConv has added support for edge_attr. I want to use this feature, but conda installed the latest pyg package, and GATConv is still consistent with the 2.0.1 version. 
What should I do? 
Thanks for the answer ！

**Comment by rusty1s** [comment, marked as answer]
Yes, it's currently only supported in master. Current plan is to release PyG `2.0.2` tomorrow.
